src/index.html

```diff
@@ -58,7 +58,10 @@
   <script src="bower_components/offline/offline.js"></script>
   <script src="bower_components/select2/dist/js/select2.js"></script>
   <script src="bower_components/underscore/underscore.js"></script>
+  <script src="bower_components/weaver-commons/dist/weaver-commons.full.js"></script>
+  <!--<script src="sdk_local/weaver-commons.full.js"></script>-->
   <script src="bower_components/weaver-sdk/dist/weaver-sdk.full.js"></script>
+  <!--<script src="sdk_local/weaver-sdk.full.js"></script>-->
   <!-- endbower -->
```

Comment:
Commented out development environment specific things should not make it onto a public git repo